ref/src/interpolate.f90

```diff
@@ -73,25 +83,26 @@ end subroutine interpolate_CalculateSpace
   !                   [o o o o o o o]
   !                   [o o o o o o o]
   !                   [x o o x o o x]
+  !
+  !*****************************************************************
 
   subroutine interpolate_array(oa, odims, f, fdims, interpFactor)
 
-
     integer, intent(in)    :: odims(4)
-    real,    intent(in)    :: oa(odims(1):odims(2), odims(3):odims(4))
+    real, intent(in)    :: oa(odims(1):odims(2), odims(3):odims(4))
     integer, intent(in)    :: fdims(4)
-    real,    intent(inout) :: f(fdims(1):fdims(2), fdims(3):fdims(4))
+    real, intent(inout) :: f(fdims(1):fdims(2), fdims(3):fdims(4))
```

Comment:
@LyndStringer Small potatoes, but the `::` are not in vertical alignment when you removed the whitespace between `real`, & `intent(in)`. Same story in line 92 above.

Reply:
Done.